pincer/client.py

```diff
@@ -162,9 +162,10 @@ def __init__(
             token,
             handlers={
                 # Use this event handler for opcode 0.
+                -1: self.payload_event_handler,

```

Comment:
This is so cursed, but i guess it works

Reply:
-1 represents infinity, so it will include everything